packages/server/src/app.ts

```diff
@@ -251,7 +251,7 @@ const loggingMiddleware = (req: Request, res: Response, next: NextFunction): voi
   const ctx = getRequestContext();
   const start = Date.now();
 
-  res.on('finish', () => {
+  res.on('close', () => {
```

Comment:
Close is actually probably what we want but it is good to note that it is slightly different than finish in exactly what we're measuring in case we see any increases in our logged request durations